Security-Team.md

```diff
@@ -45,20 +45,18 @@ includes a fix, documentation, an informational CVE or blog post.
 
 ## Team with access to private security reports against Node.js
 
-The [TSC](https://github.com/nodejs/node#tsc-technical-steering-committee)
+[TSC voting members](https://github.com/nodejs/node#tsc-voting-members)
 have access.
 
 In addition, these individuals have access:
 
-* [kumarak](https://github.com/kumarak) - **Akshay K**
-* [mike-myers-tob](https://github.com/mike-myers-tob) - **Mike Myers**
-* [RafaelGSS](https://github.com/RafaelGSS) - **Rafael Gonzaga**
-* [tn3rt](https://github.com/tn3rt) - **Trent Brunson**
+* [bengl](https://github.com/bengl)- **Bryan English**
```

Comment:
Should we add Bradley too? #1409 - FWIW We've mentioned it in today's TSC meeting

Reply:
I've invited Bradley. Once he's accepted the H1 invitation and I've added him to the list, I'll open another PR to add him (or I'll tack it on here if this is still open).